app/services/to_cocina/collection/mapper.rb

```diff
@@ -46,6 +46,12 @@ def request_params
           params_hash[:administrative][:partOfProject] = Settings.project_tag
         end
       end
+
+      def access
+        return { view: 'world' } if collection_form.license.blank?
```

Comment:
`{view: 'world', license: collection_form.license}.compact` would be more concise than L51-53